scripts/ci/tools/free-space.py

```diff
@@ -0,0 +1,43 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+"""free-space.py for cean environment before to start CI"""
+
+import subprocess
+
+exit_code = subprocess.call('./../libraries/_script_init.sh', shell=True)
+print(exit_code)
+
+class style():
+    BLUE = '\033[34m'
+
+print(style.BLUE + "Disable swap")
+subprocess.call(['sudo' , 'swapoff' , '-a'])
+subprocess.call(['sudo' , 'rm' , '-f', "/swapfile"])
+
+print(style.BLUE + "Cleaning apt")
+subprocess.call(['sudo' , 'apt' , 'clean', "||", "true"])
+
+print(style.BLUE + "Pruning docker")
+subprocess.call(["docker_v", "system", "prune", "--all", "--force", "volumes"])
```

Comment:
Yeah - we should use standard "docker" here, and as @mik-laj nicely pointed out - we should use a helper function that should print the command being executed. This way by just looking at the logs we will be able to copy&paste the command in terminal  to reproduce it.

Reply:
And also those commands should only be printed if `--verbose` is passed as an option.

Reply:
I worked on this feedback, the unique I didn't do is about docker. What I understand is that I should call, "docker" instead of "docker_v", is that correct? :)